cmd/dbseed/resolver_wrappers.go

```diff
@@ -93,6 +93,37 @@ func (s *Seeder) updatePlanBasics(
 	return updated
 }
 
+// updateDataExchangeApproach is a wrapper for resolvers.UpdatePlanDataExchangeApproach
+// It will panic if an error occurs, rather than bubbling the error up
+// It will always update the Data Exchange object with the principal value of the Model Plan's "createdBy"
+func (s *Seeder) updatePlanDataExchangeApproach(
+	// ctx context.Context,
+	// emailService oddmail.EmailService, //TODO Add these once notification is set up
+	// emailTemplateService email.TemplateService,
+	// addressBook email.AddressBook,
```

Comment:
Might be worth waiting until notification is merged since the call spec is differentiated

Reply:
Yeah, I think I'll wait for the notification PR, then update it here rather than the other way around!

Reply:
@OddTomBrooks 5e6e1959 just was pushed to resolve this. 

Currently set to hard-code email args to `nil` or `models.AddressBook{}` since we never really send emails with seed data anyways